quelpa.el

```diff
@@ -882,7 +889,11 @@ Return a cons cell whose `car' is the root and whose `cdr' is the repository."
         (when (file-exists-p dir)
           (delete-directory dir t))
         (quelpa-build--princ-checkout repo dir)
-        (quelpa-build--run-process nil "git" "clone" repo dir)))
+        (quelpa-build--run-process nil "git" "clone" repo dir)
+        (apply #'quelpa-build--run-process
```

Comment:
Note that `list` is used instead of backquoting and unquoting--this helps to avoid errors.